app/utils/callback/lava.py

```diff
@@ -383,15 +402,17 @@ def add_tests(job_data, lab_name, boot_doc_id,
         suite_data = {
             models.VERSION_KEY: "1.0",
             models.LAB_NAME_KEY: lab_name,
-            models.BOOT_ID_KEY: boot_doc_id,
             models.TIME_KEY: "0.0",
             models.NAME_KEY: suite_name,
         }
 
         utils.LOG.info("Processing test suite: {}".format(suite_name))
 
         try:
+            _get_job_meta(suite_data, job_data)
             _get_definition_meta(suite_data, job_data, META_DATA_MAP_TEST_SUITE)
+            _get_lava_meta(suite_data, job_data)
+            _add_boot_log(suite_data, job_data["log"], base_path, suite_name)
```

Comment:
I think the frontend will need to be updated to search for this log file.
Today, it searches for the boot_id and gets the log from the boot document.

Reply:
The boot log will still be generated as before, and available in the "Boots" section on the front-end.

The extra boot fields from the test entries as well as the boot log generated by the test handler are not yet used by the front-end, but this is the next step: improving test results visualisation.

It's necessary to create the logs twice at the moment to make the tests completely separate from boots.  The idea would be to stop phase out and stop using boot entries but only use tests at some point.

Reply:
OK, I understand your point. All I am saying is that the frontend is searching for "boot_id" today in test entries. From there it looks for the boot log. 

boot_id being removed the frontend will need to be updated to use this new log. Which is actually better than reusing the same log.

Reply:
Ah sorry, you meant in the Tests views - right.  As long as both the front-end and back-end are deployed in sync it should be fine I guess.  @MiloCasagrande, please let us know if you think there are any issues with doing this.

@loysollivier Are you OK with making the corresponding front-end changes?

Reply:
Yes, I will try to send a PR corresponding for the frontend tomorrow